py/torch_tensorrt/dynamo/conversion/impl/activation/base.py

```diff
@@ -4,11 +4,10 @@
 from torch.fx.node import Target
 from torch_tensorrt.dynamo._SourceIR import SourceIR
 from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
+from torch_tensorrt.dynamo.types import TRTTensor
```

Comment:
Where possible just use `tensorrt.ITensor`. eventually we want to get rid of these types